lib/iris/tests/results/COLPEX/small_colpex_theta_p_alt.cml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" ?>
 <cubes xmlns="urn:x-iris:cubeml-0.2">
-  <cube standard_name="air_potential_temperature" units="K">
+  <cube core-dtype="float32" dtype="float32" fill_value="-1.07374e+09" standard_name="air_potential_temperature" units="K">
```

Comment:
Put it this way, I can see @dkillick point, but perhaps we should change that sort order at a later date, not here ... there's way to much change going on in this PR without confusing matters further ... does that sound reasonable?